.github/workflows/releaser-signoz.yaml

```
@@ -0,0 +1,66 @@
+          path: .primus
+          token: ${{ steps.token.outputs.token }}
+      - name: info
+        run: |
+          $MAKE info
+      - name: releaser-signoz
+        run: |
+          $MAKE releaser-signoz RELEASER_ARGS="-v ${{ inputs.VERSION }} -d ${{ inputs.SIGNOZ_ROOT_DIR }}"
+      - name: create-pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2016:info:3:15: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)